docs/Developers/Development_Setup.rst

```diff
@@ -33,7 +33,7 @@ Using venv
         (coala-venv)$ deactivate
 
 - Now you can activate the environment and start
-  `the next part <https://coala.io/devsetup#installing-from-git>`_.
+  `the next part <https://api.coala.io/en/latest/Developers/Development_Setup.html#id1>`_.
```

Comment:
And actually, it's linking to this document again :O

Reply:
With \#id1 it is linking to the required link. Could you please explain why we do not need it.

Reply:
ah sorry, last time it didn't jump to the paragraph in my broswer (probably I was too impatient and scrolled). Nvm :+1: